input/capital-anonymized.md

```diff
@@ -0,0 +1,20 @@
+* The infrastructure that enables clients of the project to use the project's services.
+* A portal that allows clients and members of the public to see the accounting results and outstanding balances owed.
+* The programming language that the project is implemented in.
+* The infrastructure that runs tests and other code quality checks during development.
+* The platform publicly hosting the source code repository.
+* The version control system that records project development.
+* A code linting tool, which is used to detect simple errors during development.
+* A test coverage monitoring service, which tracks changes in code coverage over time during development.
+* A compatibility testing tool that tests the code on many different versions of the language during development.
```

Comment:
I don't know about this one, I don't think we are actually using it or that it has helped development in any way.